### ISSUE TITLE
Cannot find module 'eslint/lib/util/traverser' using npx

### ISSUE THREAD

**Issue by JaydenHarrod** [pahen/madge#194]
Hi, my team had a problem using madge@3.4.0 with npx.

Command ran:
```
npx madge --circular src/index.js
npx: installed 121 in 5.2s
Cannot find module 'eslint/lib/util/traverser'
```

To solve it, we are now using yarn/npm to run it directly from node_modules. If anyone else has this problem, just use yarn or npm instead!

Regards, Jayden

**Comment by herodrigues** [comment]
Thanks a lot!
I had this same problem.

**Comment by herodrigues** [comment]
I also have this same issue when globally installing madge using npm

**Comment by pahen** [comment]
This was caused by a change in https://github.com/eslint/typescript-eslint-parser that requires eslint to be installed as peer dependency now. I released Madge v3.4.2 that fixes this now :)
